src/tsaug/augmentor.py

```diff
@@ -1,15 +1,24 @@
+from typing import List, Tuple, Union, Optional, Callable, Any
+
 import numpy as np
 
 
 class _Augmentor:
-    def __init__(self, augmentor_func=None, is_random=None, **kwargs):
+    def __init__(
+        self,
+        augmentor_func: Optional[Callable[..., np.ndarray]] = None,
```

Comment:
This is a problem in my previous code. I think it makes more sense to set the argument `augmentor_func` as required.

Reply:
fixed